sci-electronics/kicad-packages3d/kicad-packages3d-5.0.0.ebuild

```diff
@@ -15,7 +15,7 @@ KEYWORDS="~amd64"
 IUSE=""
 
 DEPEND=""
-RDEPEND=">=sci-electronics/kicad-5.0.0[oce]"
```

Comment:
I don't think installing kicad-packages3d makes sense if kicad is compiled without occ/oce support. Maybe we should update it to RDEPEND on either of those flags enabled for kicad.

Reply:
Would it make sense to add the oce and occ USE flags to the kicad-meta ebuild? This is where this package gets pulled in in the first place.

Reply:
Not really, kicad-meta is just convenience "package" for users to pull in everything. One could cherry pick what they install (e.g. they use their own libraries and don't pull in the Gentoo provided ones) and in that case their installation would be broken since kicad could be compiled without OCC/OCE support and thus packages3d contents would not work.